python-package/xgboost/tracker.py

```diff
@@ -293,7 +293,7 @@ def accept_slaves(self, nslave):
             s = SlaveEntry(fd, s_addr)
             if s.cmd == 'print':
                 msg = s.sock.recvstr()
-                logging.info(msg.strip())
```

Comment:
The `logging` module should be preferred over the simple `print`, since Dask has a good integration with the `logging` module. See related discussion at https://github.com/rapidsai/cuml/issues/3350.

Reply:
@hcho3 I definitely want to use logging.  But what logging level should be used?  To print out the evaluation history, it should not be info, as the default in Python logging is warning.  But also the evaluation history should not be warning right?

Reply:
Quoting from https://docs.python.org/3/howto/logging.html :


Task you want to perform | The best tool for the task
-- | --
Display console output for ordinary usage of a command line script or program | print()
Report events that occur during normal operation of a program (e.g. for status monitoring or fault investigation) | logging.info() (or logging.debug() for very detailed output for diagnostic purposes)
Issue a warning regarding a particular runtime event | warnings.warn() in library code if the issue is avoidable and the client application should be modified to eliminate the warning logging.warning() if there is nothing the client application can do about the situation, but the event should still be noted
Report an error regarding a particular runtime event | Raise an exception
Report suppression of an error without raising an exception (e.g. error handler in a long-running server process) | logging.error(), logging.exception() or logging.critical() as appropriate for the specific error and application domain

I think simple print is a reasonable choice here.

Reply:
@hcho3 

I don't think we should print something like this:

```
INFO:root:test-data:error:2.71 train-data:error:2.33
...
```

Reply:
Got it. Thanks for the reference.